pkg/resource/pvc/scanner.go

```diff
@@ -33,7 +35,16 @@ func (s *Scanner) ID() resource.ScannerID {
 }
 
 func (s *Scanner) Scan(ctx context.Context, runtime *runtime.Info) (resource.ScanConverter, error) {
-	ctx, span := otel.Tracer("").Start(ctx, "kmc.pvc_scan")
+	ctx, span := otel.Tracer("").Start(ctx, "kmc.pvc_scan",
+		trace.WithAttributes(
+			attribute.String("instance_id", runtime.InstanceID),
+			attribute.String("runtime_id", runtime.RuntimeID),
+			attribute.String("sub_account_id", runtime.SubAccountID),
+			attribute.String("global_account_id", runtime.GlobalAccountID),
+			attribute.String("shoot_name", runtime.ShootName),
+			attribute.String("provider", runtime.ProviderType),
+		),
+	)
```

Comment:
please extract a reusable function, as every scanner will need this

Reply:
We have now a reusable function `SpanAttributes` in the otel package which can be used by all scanners and collector